Source/ChocolateyGui.Common.Windows/Bootstrapper.cs

```diff
@@ -121,6 +121,7 @@ protected override async void OnStartup(object sender, StartupEventArgs e)
 
                 var packageService = Container.Resolve<IChocolateyService>();
                 var features = await packageService.GetFeatures();
+                var fileSystemWatcher = Container.Resolve<IConfigFileWatcher>();

```

Comment:
This is being resolved, but isn't being used, or injected anywhere, it that intentional?

Reply:
In my testing, it seemed that the watcher wasn't being instantiated until it was resolved, and it seemed to make sense to resolve it where all of the other services were being resolved.

Would it be better to resolve it when we add it to the container?

Reply:
We spoke about this, and it's all good.